packages/app-bridge/src/AppBridgeTheme.ts

```diff
@@ -251,4 +257,20 @@ export class AppBridgeTheme {
     public async getColorsByColorPaletteId(colorPaletteId: number): Promise<Color[]> {
         return getColorsByColorPaletteId(colorPaletteId);
     }
+
+    public getDocumentTargets = (id: number): Promise<Targets> => {
```

Comment:
```suggestion
    public async getDocumentTargets(id: number): Promise<Targets> {
```
Same for the other properties you added